oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -81,9 +85,7 @@ protected int getDefaultPort() {
     return 1521;
   }
 
-  public String getConnectionType() {
-    return connectionType;
-  }
+  public String getConnectionType() {return connectionType == null ? "service" : connectionType;}
```

Comment:
connectionType should not be null. Did you observe it as `null` in any of your testing?

Reply:
I did not. It was always populated.